modules/button/react/lib/ToolbarDropdownButton.tsx

```diff
@@ -0,0 +1,88 @@
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import styled from '@emotion/styled';
+import {colors, spacing, borderRadius} from '@workday/canvas-kit-react-core';
+import {focusRing, useTheme, Themeable, EmotionCanvasTheme} from '@workday/canvas-kit-react-common';
+import {SystemIcon} from '@workday/canvas-kit-react-icon';
+import {ButtonColors} from './types';
+import {ButtonContainer} from './parts';
+import {IconButtonProps} from './IconButton';
+import {chevronDownSmallIcon} from '@workday/canvas-system-icons-web';
+
+export interface ToolbarDropdownButtonProps
+  extends Omit<IconButtonProps, 'size' | 'variant' | 'toggled' | 'onToggleChange'>,
```

Comment:
Nit: This will be cleaner if you extend `ToolbarIconButtonProps`

Reply:
Also, are we certain there is no use case for toggling these buttons? If there were, I would rather this just be a `dropdown?: boolean` prop in `ToolbarIconButton`

Reply:
I've talked to design and this button is not toggleable, so I just extended the props from toolbarIconButton which did clean up the omits.